_test/test_mpi/finish_task_tester.m

```diff
@@ -1,4 +1,4 @@
-function ok=finish_task_tester(worker_controls_string)
+function ok=finish_task_tester(worker_controls_string,varargin)
 %Inputs:
 % worker_controls_string - the structure, containing information, necessary to
```

Comment:
missing documentation of the optional varargs

Reply:
Done, not sure how understandable this description is -- the models used in testing of MPI communications without running real MPI job is not fully straightforward.